app/services/user_facing_state.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class UserFacingState
+  attr_reader :document
+  delegate :review_state, :publication_state, to: :document
+
+  def initialize(document)
+    @document = document
+  end
+
+  def to_s
+    if review_state == "submitted_for_review"
+      "submitted_for_review"
+    elsif publication_state.in?(%w[changes_not_sent_to_draft sent_to_draft sending_to_draft error_sending_to_draft sending_to_live])
```

Comment:
Wondering if having `error_sending_to_draft` here will cause problems ? would it be easier for now to have a user facing error state to prevent other actions on the document e.g. to stop someone 'publishing' something which hasn't been sent to draft yet.

Reply:
That's a really good point. I've put in a story here https://trello.com/c/VkVrVGeB, because this is something Ben and Sonia need to think about. I'll pick that up directly after this.